scripts/binary/trigger-publish-binary-pipeline.js

```diff
@@ -16,7 +16,7 @@ const { getNextVersionForBinary } = require('../get-next-version')
       job_name: process.env.CIRCLE_JOB,
       triggered_workflow_id: process.env.CIRCLE_WORKFLOW_ID,
       triggered_job_url: process.env.CIRCLE_BUILD_URL,
-      branch: process.env.CIRCLE_BRANCH,
+      branch: 'upgrade-electron-28',
```

Comment:
```suggestion
      branch: 'upgrade-electron-29',
```

This will need another PR